SpecialRequestWiki.php

```diff
@@ -151,3 +151,11 @@ function handleRequestWikiFormInput() {
 		$this->getOutput()->addHTML( '<div class="successbox">' . $this->msg( 'requestwiki-success', $idlink )->plain() . '</div>' );
 	}
 }
+
+function has_hyphens($string)
+{
+    return preg_match('/[\-]/', $string);
+}
+
+if(has_hyphens('-'))
```

Comment:
I say this because if i understand that function I believe if(has_hyphens('-')) would always return true and thus always throw new exceptions